j2cl-maven-plugin/src/it/translation/src/main/java/org/treblereel/test/holder.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright © 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+goog.provide('holder');
+/** @define {string} */
+holder.value = goog.define('holder.value', 'unknown');
```

Comment:
trailing newline, copyright notice not required (following current repo conventions anyway - i might be wrong about it, but i'm not going to object if this file is copied without permission).